Make Python wrapper works based on commit 60ecf9e.
diff --git a/Makefile b/Makefile
index 19bfcae..3c76292 100644
--- a/Makefile
+++ b/Makefile
@@ -93,7 +93,7 @@ $(ALIB): $(OBJS)
 	$(AR) $(ARFLAGS) $@ $^
 
 $(SLIB): $(OBJS)
-	$(CC) $(CFLAGS) -shared $^ -o $@
+	$(CC) $(CFLAGS) -shared $^ -o $@ $(LDFLAGS)
 
 $(OBJDIR)%.o: %.c $(DEPS)
 	$(CC) $(COMMON) $(CFLAGS) -c $< -o $@
diff --git a/scripts/gen_tactic.sh b/scripts/gen_tactic.sh
old mode 100755
new mode 100644
diff --git a/src/network.c b/src/network.c
index 6425855..73ffad5 100644
--- a/src/network.c
+++ b/src/network.c
@@ -1,3 +1,4 @@
+#include <stdint.h>
 #include <stdio.h>
 #include <time.h>
 #include <assert.h>
@@ -55,6 +56,10 @@ network *load_network(char *cfg, char *weights, int clear)
         load_weights(net, weights);
     }
     if(clear) (*net->seen) = 0;
+#ifdef NNPACK
+    nnp_initialize();
+    net->threadpool = pthreadpool_create(4);
+#endif
     return net;
 }
 
